packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -356,8 +357,9 @@ export function generateGrid(
                         sortable: !sortFields.includes(column.name) ? `false` : undefined,
                         valueGetter: column.valueGetter,
                         valueOptions: column.valueOptions,
-                        width: column.width ? String(column.width) : "150",
                         renderCell: column.renderCell,
+                        width: typeof column.width === "undefined" ? undefined : String(column.width),
+                        flex: typeof column.width === "undefined" ? "1" : undefined,
```

Comment:
Is this default value a good idea? Couldn't we add something like `column.width: "auto"`?

Reply:
could we have no defaults and let devs use the mui docs
https://mui.com/x/react-data-grid/column-dimensions/?

Reply:
We'd then need to add `flex` to the column definition in the codegen. Fine by me. We could even add a link to the MUI docs to the option. In my experience devs aren't aware of the MUI docs, sadly...

Reply:
> In my experience devs aren't aware of the MUI docs, sadly...

what???

Reply:
> what???

To be more concrete: They may know that we use MUI and the docs exist, but the MUI docs aren't their first place to go to. They'd rather ask a colleague or copy it from another project. Both is fine, but discovering the solution in the docs themselves is preferable.

Reply:
I think this "auto" behavior should be the default, possibly with a `minWidth`. 
This would avoid needing additional configuration for most cases, see: https://github.com/vivid-planet/comet/pull/1784#issuecomment-1970728212